api/bucket-handlers.go

```diff
@@ -255,9 +255,9 @@ func (api ObjectAPIHandlers) ListBucketsHandler(w http.ResponseWriter, r *http.R
 
 // DeleteMultipleObjectsHandler - deletes multiple objects.
 func (api ObjectAPIHandlers) DeleteMultipleObjectsHandler(w http.ResponseWriter, r *http.Request) {
+	ctx := getRequestContext(r)
```

Comment:
I'd like to reserve `ctx` for `context.Context` objects, make it `reqCtx` or `reqContext`, personal opnion.